source/slang/slang-check-expr.cpp

```diff
@@ -3815,6 +3861,108 @@ namespace Slang
         return CreateErrorExpr(memberRefExpr);
     }
 
+    Expr* SemanticsVisitor::checkTupleSwizzleExpr(MemberExpr* memberExpr, TupleType* baseTupleType)
```

Comment:
The reason here is similar to vector/matrix swizzles in that if we see `v.xyz` during parsing, since we haven't checked the type for `v` yet (this is done during semantic checking), we don't know if this is a swizzle or just a normal lookup. We only know it is a swizzle after checking `v`, and at that time we will have to parse the member string again to get the swizzle structure.